I-D_in_xml/draft-ietf-ccamp-optical-impairment-topology-yang-16.txt

```diff
@@ -0,0 +1,8232 @@
+      |  |  +----|   |   /   |   |  \   |    o------/    | 1  o------o->
+    --o->|  | Tx o-->o--o  5 |   |   ---o  4 |         /-o    |   |  |
+      |  |  +----|   |   \   |  AD2     |    o-\      /  |___/    |  |
+    <-o--|  | Rx o   |    ---o-->o--     \___|  \    /        DEG1|  |
+      |  |  +----|   +-------+   |  \     ___    \   |    ___     |  |
+      |  |       |               |   \   /   |    \  |   |   \  Line |
+      |  +-------+      internal |    \ |    o-------/   o    | LTP 2|
+      |                 AD ports |     -o  6 |      \    | 2  o------o->
+      |                          |      |    o-------\---o    |   |  |
+      |                          o       \___|        |  |___/    |  |
+      |                          |                    |       DEG2|  |
+      |                          |                    |   ___     |  |
+      |                          |                    |  |   \  Line |
+      |                          |                    \--o    | LTP 3|
+      |                          |                       | 3  o------o->
+      |                          o                       o    |   |  |
+      |                          |                       |___/    |  |
+      |                          |                            DEG3|  |
+      |                          +--------------------------------+  |
+      +--------------------------------------------------------------+
+
+
+       Figure 29: Protected TTP with Restricted Connectivity between
+                ROADM Add-Drop Ports and ROADM Degree Ports
+
+   {
+     "roadm-path-impairments": [
+       {
+         "roadm-path-impairments-id": 1,
+         "roadm-add-path":
+         "Add path impairments from TTP 1 or TTP 2 to any LTPs."
+       },
+       {
+         "roadm-path-impairments-id": 2,
+         "roadm-add-path": "Add path impairments from TTP 3 or TTP 4
+          to LTP 1 or LTP 3, through AD1."
+       },
+       {
+         "roadm-path-impairments-id": 3,
+
+
+
+Beller, et al.           Expires 9 January 2025               [Page 107]
+
+Internet-Draft    Opt. Impairment-Aware Topo YANG Model        July 2024
+
+
+         "roadm-add-path": "Add path impairments from TTP 3 or TTP 4
+          to LTP 1 or LTP 2, through AD2."
+       }
+     ],
+     "tunnel-termination-point": [
+       {
+         "tunnel-tp-id": 1,
+         "protection-type": "ops-protection",
+         "local-link-connectivities": {
+           "is-allowed": true,
+           "add-path-impairments": 1
+         }
+       },
+       {
+         "tunnel-tp-id": 2,
+         "local-link-connectivities": {
+           "is-allowed": true,
+           "add-path-impairments": 1,
+           "local-link-connectivity": [
+             {
+               "link-tp-ref": "LTP-1",
+               "additional-ltp": [
+                 {
+                   "link-tp-ref": "LTP-2",
+                   "link-tp-ref": "LTP-3"
```

Comment:
json is nal formed: the list can not contain the same key multiple times, and its name has been changed to  ltp-ref

Reply:
@EstherLerouzic : correct , we will fix